pkg/background/update_request_controller.go

```diff
@@ -204,7 +204,9 @@ func (c *Controller) syncUpdateRequest(key string) error {
 
 	logger.V(3).Info("UR is marked successfully", "ur", ur.GetName(), "resourceVersion", ur.GetResourceVersion())
 	if err := c.ProcessUR(ur); err != nil {
-		return fmt.Errorf("failed to process UR %s: %v", key, err)
+		logger.V(2).Info("failed to process the UR", "handler", ur.Status.Handler, "err", err)
```

Comment:
What are the scenarios where we want to delete URs and target resources? I don't think we want to do so on failure for every reconciliation.

Reply:
It will not delete utils the trigger resource is deleted ,its a similar behaviour as cleanup controller